src/vmm/src/rpc_interface.rs

```diff
@@ -32,9 +32,9 @@ use crate::vmm_config::net::{
 use crate::vmm_config::snapshot::{CreateSnapshotParams, LoadSnapshotParams, SnapshotType};
 use crate::vmm_config::vsock::{VsockConfigError, VsockDeviceConfig};
 use crate::vmm_config::{self, RateLimiterUpdate};
+use crate::FcExitCode;
 use crate::{builder::StartMicrovmError, warn, EventManager};
-use crate::{ExitCode, FC_EXIT_CODE_BAD_CONFIGURATION};
-use logger::{error, info, update_metric_with_elapsed_time, METRICS};
+use logger::{info, update_metric_with_elapsed_time, METRICS};
```

Comment:
error is still used in this file at line 352. At line 353 there is also a reference to FC_EXIT_CODE_GENERIC_ERROR which does not exist anymore.

Reply:
I've cleaned up the branch and I beleive fixed this import issue.